Zend/zend_constants.c

```diff
@@ -583,3 +567,9 @@ ZEND_API zend_result zend_register_constant(zend_constant *c)
 	}
 	return ret;
 }
+
+ZEND_API zend_result zend_register_internal_constant(const char *name, size_t name_len, zend_constant *c) {
+	zend_string *name_str = zend_string_init_interned(name, name_len, ZEND_CONSTANT_FLAGS(c) & CONST_PERSISTENT);
+
+	return zend_register_constant(name_str, c);
```

Comment:
You probably need to release "name_str" in case of failure, or may be even in case of success.

Reply:
Yes, thanks I completely forgot about handling the error case.. :/ Unfortunately, somehow I cannot reproduce any of the memleaks locally, so it's difficult to find where the issue is.

Reply:
Although... `zend_string_release()` doesn't have an effect on interned strings.. So I guess it doesn't help much (or I should use a different function)